### PR TITLE
create initialize function

### PR DESCRIPTION
**Pull request by 4geru**
# DONE
- [x] create init function
- [x] update help command
- [x] update README.md

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -17,6 +17,8 @@
     "liff": "./main.js"
   },
   "dependencies": {
+    "dotenv": "^6.0.0",
+    "fs": "0.0.1-security",
```
Comment (morugu) [inline package.json]: Why is `fs` necessary?

File: README.md [hunk 2]
```diff
@@ -6,10 +6,16 @@ https://developers.line.me/en/docs/liff/overview/
 # Installation
 ```
 $ npm install -g liff
-$ export LINE_ACCESS_TOKEN={YOUR_LINE_ACCESS_TOKEN}
+$ liff init _line_access_token_
```
Comment (morugu) [inline README.md]: I think that `{LINE_ACCESS_TOKEN}` better. explicit message.

File: package.json (hunk 1 above)
Comment (4geru) [reply package.json]: I thought need to write file. but it don't need it :)

**Comment by morugu** [comment]
LGTM 👍

**Pull request merged**